apps/docs/content/components/List.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: List
```

Comment:
Can we have two different doc pages for each component instead of one. `List` isn't a component so this is somewhat confusing.